src/include/sof/audio/module_adapter/module/generic.h

```diff
@@ -203,6 +205,11 @@ struct processing_module {
 	 */
 	bool skip_src_buffer_invalidate;
 
+	/* False for module with one sink and one source to reduce module processing
+	 * overhead. True if multiple source or sink buffers.
+	 */
+	bool multi_source_or_sink;
```

Comment:
bool simple_1_1_copy ?
to be consistent with module_adapter_audio_stream_copy_1to1 that is called when true